.github/workflows/tests.yml

```diff
@@ -357,8 +364,8 @@ jobs:
       - name: Merge Artifacts
         uses: actions/upload-artifact/merge@v4
         with:
-          name: failed-system-test-screenshots
-          pattern: failed-system-test-screenshots-*
+          name: failed-system-test-artifacts
+          pattern: failed-system-test-artifacts-*
```

Comment:
This rename will break the [discourse-flaky-tests-monitor plugin](https://github.com/discourse-org/discourse-flaky-tests-monitor/blob/55af62b050cc754dd8ca8e2d73b03b8dafe15a4e/app/jobs/regular/process_github_tests_workflow_run.rb#L9). I won't have time to fix it this week but I guess it is OK for it to be broken for a little while.